universal_transfer_operator/src/universal_transfer_operator/data_providers/database/base.py

```diff
@@ -0,0 +1,659 @@
+from __future__ import annotations
```

Comment:
A significant part of the code we're adding to UTO comes from the Python SDK. I would love to see references/credits to the original code in the commit message.